server/events/markdown_renderer.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 const (
-	planCommandTitle  = "Plan"
-	applyCommandTitle = "Apply"
+	planCommandTitle    = "Plan"
+	applyCommandTitle   = "Apply"
+	discardCommandTitle = "Discard"
```

Comment:
I don't think you need to modify this renderer. Should be able to construct this comment in command_runner.

Reply:
Done. I left the bit that shows the option for discarding plans

Reply:
I mean after a successful plan, you get options to click to unlock, apply etc, so I added the unlock option